winterfell/utils/formProperties.js

```diff
@@ -0,0 +1,54 @@
+function FormProperties(app) {
+    this.app = app;
+}
+
+module.exports = FormProperties;
+
+module.exports.FORM_ID = {
+  INTENT_TO_FILE_CLAIM: 'vba_21_0966_are'
```

Comment:
ok I'll label a `static` folder which has static configurations shared by both backend and frontend.